src/locales/zh/translation.json

```diff
@@ -9,6 +9,10 @@
   "high_risk.title": "高危地區",
   "high_risk.source": "資料來源：衞生防護中心",
   "hygiene_tips.title": "自保貼士",
+  "poster_gallery.title": "文宣圖集",
+  "poster_gallery.source": "資料來源：反送中運動文宣圖集",
```

Comment:
@himphen Do you mind if you change that to 資料來源：Collaction?

Reply:
@cswbrian I suggest using 反送中運動文宣圖集 which is better since the website is a crowdsourcing project contributed by everyone. And also we will move it to a standalone website later 😂